### PR TITLE
Add react native API examples to the Target AEP docs

### PR DESCRIPTION
**Pull request by rymorale**

### CODE REVIEW

File: using-mobile-extensions/adobe-target/target-api-reference.md [hunk 1]
```diff
@@ -706,6 +836,58 @@ let targetParameters = ACPTargetParameters(parameters: mboxParameters, profilePa
 ACPTarget.retrieveLocationContent(requestArray, with: targetParameters)
 ```
 {% endtab %}
+
+{% tab title="React Native" %}
+
+### retrieveLocationContent
+
+This API will send a batch request to configured Target server only for mbox locations in the `TargetRequest` list that are not already prefetched. The content for the mbox locations which are already prefetched with a prior prefetch request, will be returned from within the SDK itself. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
```
[the remark this answers is not in the document]
Comment (rymorale) [reply using-mobile-extensions/adobe-target/target-api-reference.md]: I reworded the second sentence as "The content for the mbox locations that have been prefetched in a previous request are returned from within the SDK and no additional network request is made." for some added clarity.

[the remark this answers is not in the document]
Comment (rekrishn) [reply using-mobile-extensions/adobe-target/target-api-reference.md]: @rymorale, thanks for this update...I've tweaked the wording a little bit:
""The content for the mbox locations that have been prefetched in a previous request are returned from the SDK and no additional network request is made." 

Let me know what you think.

File: using-mobile-extensions/adobe-target/target-api-reference.md [hunk 2]
```diff
@@ -1343,5 +1572,96 @@ The following method can be used to create an instance of ACPTargetProduct.
...
+  	this.orderId = orderId;
+    this.total = total;
+    this.purchasedProductIds = purchasedProductIds;
+  }
+}
+```
+
+### ACPTargetProduct
+
+This class contains productId and categoryId.
```
[the remark this answers is not in the document]
Comment (rymorale) [reply using-mobile-extensions/adobe-target/target-api-reference.md]: I think its `Id` as its referencing the variable names.

File: using-mobile-extensions/adobe-target/target-api-reference.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rymorale) [reply using-mobile-extensions/adobe-target/target-api-reference.md]: Yup, this sounds better. Thanks!

File: using-mobile-extensions/adobe-target/README.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 
 {% tab title="React Native" %}
 
-To register the Target with Core, use the following API:
+To register the Target extension with the MobileCore Extension, use the following API:
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/README.md]: - please make "Extension" lower case --> "extension"
- Can we make "MobileCore" 2 word? --> "Mobile Core" (that's how i've typically seen it.

File: using-mobile-extensions/adobe-target/target-api-reference.md [hunk 4]
```diff
@@ -664,7 +664,7 @@ Use this API to send a batch request to the configured Target server for multipl
 {% tab title="Android" %}
 ### retrieveLocationContent
 
-This API will send a batch request to configured Target server only for mbox locations in the `TargetRequest` list that are not already prefetched. The content for the mbox locations which are already prefetched with a prior prefetch request, will be returned from within the SDK itself. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
+For mbox locations in the `TargetRequest` list that are not already prefetched, this API sends a batch request to the configured Target server. The content for the mbox locations that have been prefetched in a previous request are returned from the SDK and no additional network request is made. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/target-api-reference.md]: ah, my bad...in the 2nd sentence, please add a comma between "SDK" and "and"

File: using-mobile-extensions/adobe-target/target-api-reference.md [hunk 5]
```diff
@@ -732,7 +732,7 @@ Target.retrieveLocationContent(locationRequests, parameters);
 {% tab title="iOS" %}
 ### retrieveLocationContent
 
-This API will send a batch request to configured Target server only for mbox locations in the `TargetRequest` list that are not already prefetched. The content for the mbox locations which are already prefetched with a prior prefetch request, will be returned from within the SDK itself. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
+For mbox locations in the `TargetRequest` list that are not already prefetched, this API sends a batch request to the configured Target server. The content for the mbox locations that have been prefetched in a previous request are returned from the SDK and no additional network request is made. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/target-api-reference.md]: in the 2nd sentence, please add a comma between "SDK" and "and"

File: using-mobile-extensions/adobe-target/target-api-reference.md [hunk 6]
```diff
@@ -841,7 +841,7 @@ ACPTarget.retrieveLocationContent(requestArray, with: targetParameters)
 
 ### retrieveLocationContent
 
-This API will send a batch request to configured Target server only for mbox locations in the `TargetRequest` list that are not already prefetched. The content for the mbox locations which are already prefetched with a prior prefetch request, will be returned from within the SDK itself. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
+For mbox locations in the `TargetRequest` list that are not already prefetched, this API sends a batch request to the configured Target server. The content for the mbox locations that have been prefetched in a previous request are returned from the SDK and no additional network request is made. Each `TargetRequest` object in the list contains a callback function, which is invoked when content is available for its given mbox location.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-target/target-api-reference.md]: in the 2nd sentence, please add a comma between "SDK" and "and"

Comment (rymorale) [reply using-mobile-extensions/adobe-target/target-api-reference.md]: I added the missing commas in this commit: https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/207/commits/4534de311a084c6f00d938ea0dcda6e650b39b62

Comment (rymorale) [reply using-mobile-extensions/adobe-target/target-api-reference.md]: I added the missing commas in this commit: https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/207/commits/4534de311a084c6f00d938ea0dcda6e650b39b62

**Pull request merged**